recipes-wpe/wpe/wpe_0.1.bb

```diff
@@ -186,4 +186,7 @@ RDEPS_MEDIASOURCE += "${RDEPS_EXTRA}"
 RDEPS_VIDEO += "${RDEPS_EXTRA}"
 RDEPS_WEBAUDIO += "${RDEPS_EXTRA}"
 
-RRECOMMENDS_${PN} += "ca-certificates"
+RRECOMMENDS_${PN} += " \
+    ca-certificates \
+    ttf-bitstream-vera \
```

Comment:
The browser doesn't work if you don't have any fonts, so leaving fonts as the responsibility of the image gives a bad out of box experience for anyone who doesn't know that. If Comcast want to use different fonts then that's fine. You can add BAD_RECOMMENDATIONS = "ttf-bitstream-vera" in your distro.